backend/services/mgmt/v1alpha1/connection-data-service/connection-data.go

```diff
@@ -637,6 +637,9 @@ func (s *Service) GetConnectionSchema(
 				if err != nil {
 					return nil, err
 				}
+				if result.ContentLength == nil || *result.ContentLength == 0 {
+					continue
```

Comment:
Same, is it worth adding a log statement here? It might be helpful for debugging if this is an edge case.